hardware-handler/nightmare_movement_engine/src/main.py

```diff
@@ -26,9 +30,10 @@ class engineNode():
     def __init__(self):
```

Comment:
Rename to `EngineNode` and pass these values to the constructor instead of creating them here

Reply:
fixed